help/en/Acknowledgements.shtml

```diff
@@ -126,8 +126,7 @@
         <li>Leo Bicknell, who updated the LocoNet monitor</li>
 
         <li><a href="https://github.com/JMRI/JMRI/pulls?q=author%3Albijlsma+">Lolke Bijlsma</a>, <!-- lbijlsma  -->
-        who provided a decoder definition for the Shine Plus decoder series from
-        Train-O-Rama.</li>
+        who provided a decoder definitions for Train-O-Rama decoders.</li>
```

Comment:
I'm not a native English speaker, but I believe the word "a" should be removed since "definitions" is plural.

who provided decoder definitions ...

Reply:
I'm not a native speaker either but you are 100% correct. I'll submit another PR to address this.